lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -395,87 +395,6 @@
             color: $twitter-blue;
           }
         }
-
```

Comment:
I originally had the styles here, but decided to break them out into their own partial for easier management.